pom.xml

```diff
@@ -138,7 +138,7 @@
     <http.version>4.4.1</http.version>
     <spark.version>${spark3.version}</spark.version>
     <spark2.version>2.4.4</spark2.version>
-    <spark3.version>3.4.1</spark3.version>
+    <spark3.version>3.4.3</spark3.version>
```

Comment:
This should use `3.5.1`, the latest supported Spark version.

Reply:
Fixed. Thank you.